sunpy/net/tests/test_hek2vso.py

```diff
@@ -61,28 +64,98 @@ def test_translate_results_to_query():
 
 
 @pytest.mark.remote_data
-def test_vso_attribute_parse():
+def test_vso_attribute_parse(hek_client):
     """Make sure that Parsing of VSO attributes from HEK queries is accurate"""
-    h = hek.HEKClient()
+    h = hek_client
     hek_query = h.search(hekTime, hekEvent)
     vso_query = hek2vso.vso_attribute_parse(hek_query[0])
 
     # Checking Time
-    assert vso_query[0].start == parse_time(hek_query[0]['event_starttime'])
-    assert vso_query[0].end == parse_time(hek_query[0]['event_endtime'])
+    assert vso_query[0].start == parse_time(hek_query[0]["event_starttime"])
+    assert vso_query[0].end == parse_time(hek_query[0]["event_endtime"])
 
     # Checking Observatory
-    assert vso_query[1].value == hek_query[0]['obs_observatory']
+    assert vso_query[1].value == hek_query[0]["obs_observatory"]
 
     # Checking Instrument
-    assert vso_query[2].value == hek_query[0]['obs_instrument']
+    assert vso_query[2].value == hek_query[0]["obs_instrument"]
 
     # Checking Wavelength
-    assert vso_query[3].min == hek_query[0]['obs_meanwavel'] * u.Unit(hek_query[0]['obs_wavelunit'])
-    assert vso_query[3].max == hek_query[0]['obs_meanwavel'] * u.Unit(hek_query[0]['obs_wavelunit'])
-    assert vso_query[3].unit == u.Unit('Angstrom')
+    assert vso_query[3].min == hek_query[0]["obs_meanwavel"] * u.Unit(
+        hek_query[0]["obs_wavelunit"]
+    )
+    assert vso_query[3].max == hek_query[0]["obs_meanwavel"] * u.Unit(
+        hek_query[0]["obs_wavelunit"]
+    )
+    assert vso_query[3].unit == u.Unit("Angstrom")
 
 
+@pytest.mark.remote_data
 class TestH2VClient(object):
```

Comment:
Can you undo this class structure?

Reply:
Means ?

Reply:
Can you rewrite the tests to not use this class?

Reply:
Yup. I can have the functions written in the class separately right ?

Reply:
You can remove the functions from the class sure.

Reply:
Like this ?

Reply:
Yes